src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -643,10 +645,10 @@ void LedgerConsensusImp::handleLCL (uint256 const& lclHash)
             // Tell the ledger acquire system that we need the consensus ledger
             mAcquiringLedger = mPrevLedgerHash;
             auto& previousHash = mPrevLedgerHash;
-            getApp().getJobQueue().addJob (
+            app_.getJobQueue().addJob (
                 jtADVANCE, "getConsensusLedger",
-                [previousHash] (Job&) {
-                    getApp().getInboundLedgers().acquire(
+                [&] (Job&) {
+                    app_.getInboundLedgers().acquire(
```

Comment:
This looks like it could potentially have the same lifetime problem with the lambda.